LICENSE

```diff
@@ -95,9 +95,9 @@ case, there is little to gain by limiting the free library to free
 software only, so we use the Lesser General Public License.
 
   In other cases, permission to use a particular library in non-free
-programs enables a greater number of people to use a large body of
+programs enable a greater number of people to use a large body of
```

Comment:
This is correct as originally written, since it's not the programs that enable something, but the **permission** to use the programs.